Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -467,4 +460,74 @@ protected function configureCacheDirectory()
 
         $this->cacheDirectory = $cacheDirectory;
     }
+
+    /**
+     * Reads the cache data from the given cache file, using locking.
+     *
+     * @param string $cacheEntryPathAndFilename
+     * @param int|null $offset
+     * @param int|null $maxlen
+     * @return boolean|string The contents of the cache file or FALSE on error
+     */
+    protected function readCacheFile($cacheEntryPathAndFilename, $offset = null, $maxlen = null)
+    {
+        for ($i = 0; $i < 3; $i++) {
+            $data = false;
+            try {
+                $file = fopen($cacheEntryPathAndFilename, 'rb');
+                if ($file === false) {
+                    continue;
```

Comment:
see below. Can we add a comment when this case can happen? IMHO it can happen when the file was removed in the meantime